prov/psm/src/psmx_init.c

```diff
@@ -558,6 +558,7 @@ static int psmx_getinfo(uint32_t version, const char *node, const char *service,
 	psmx_info->domain_attr->mr_iov_limit = 65535;
 	psmx_info->domain_attr->caps = PSMX_DOM_CAPS;
 	psmx_info->domain_attr->mode = 0;
+	psmx_info->domain_attr->mr_cnt = 10000;
```

Comment:
Number of MRs in psm and psm2 is only limited by the key, which is 64 bits. So -1ULL would work here. For practical purpose, any number that is large enough should be fine, but value like 65535 or 2097151 is more preferable than 10000.